web3/main.py

```diff
@@ -82,17 +84,17 @@
 
 
 def get_default_modules():
-    return {
-        "eth": Eth,
-        "net": Net,
-        "personal": Personal,
-        "version": Version,
-        "txpool": TxPool,
-        "miner": Miner,
-        "admin": Admin,
-        "parity": Parity,
-        "testing": Testing,
-    }
+    return [
+        {"name": "eth", "module": Eth},
+        {"name": "net", "module": Net},
+        {"name": "version", "module": Version},
+        {"name": "txpool", "module": TxPool},
+        {"name": "miner", "module": Miner},
+        {"name": "admin", "module": Admin},
+        {"name": "parity", "module": Parity, 'submodules': {'personal': ParityPersonal}},
```

Comment:
You may like this format better:
```py
return {
  ...
  "admin": Admin,
  "parity": (Parity, {"personal": ParityPersonal}),
  ...
}
```
One nice thing about this is that the submodule format is the same as the supermodule format, which makes for easier extension to multiple layers if desired. It also means less special casing when stitching the modules togother. Plus I think it's a little nicer to read.

Here's an example with 3 layers:
```py
return {
  ...
  "admin": Admin,
  "parity": (Parity, {
    "personal": ParityPersonal,
    "test": (TestModule, {
      "eth": ParityEthTest,
      "bzz": ParitySwarmTest,
      ...
    }),
  }),
  ...
}
```